### PR TITLE
Added pattern for different oc outputs

### PR DESCRIPTION
**Pull request by tjaeschke**
relates to #159

### CODE REVIEW

**Comment by stefanlack** [comment]
LGTM

**Comment by michaelsauter** [comment]
Does not work for us. Please do not merge. I'll update.

File: vars/stageStartOpenshiftBuild.groovy [hunk 1]
```diff
@@ -15,7 +15,12 @@ def call(def context, def buildArgs = [:], def imageLabels = [:]) {
         returnStdout: true
       ).trim()
       echo startBuildInfo
-      buildId = startBuildInfo.split(' ').first().split('/').last()
+      def match = (startBuildInfo =~ /\"[a-z_A-Z0-9\s-\/]*\"/)
+      if(match.find()) {
+        buildId = match.group(0).replace('"','')
+      } else {
+        buildId = startBuildInfo.split(' ').first().split('/').last()
+      }
```
Comment (michaelsauter) [inline vars/stageStartOpenshiftBuild.groovy]:
```suggestion
def startBuildInfoHeader = startBuildInfo.readLines().first()
def match = (startBuildInfoHeader =~ /${context.componentId}-([0-9]+)/)
if (match.find()) {
    buildId = res[0][1]
} else {
  error "Did not find build ID in build start result '${startBuildInfo}'"
}
```

**Comment by stefanlack** [comment]
@michaelsauter can you please give an example of the content of _startBuildInfo_ in your enviroment, so we can test both versions?

This is the contend on our site:
`String startBuildInfo = 'build "prov-app-1" started'`

**Comment by michaelsauter** [comment]
One sec

**Comment by michaelsauter** [comment]
@stefanlack It is `build.build.openshift.io/be-main-667 started`. I noticed though that `readLines` results in `Scripts not permitted to use staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods readLines java.lang.String`, I'm testing an alternative and will update.

**Comment by michaelsauter** [comment]
@stefanlack @tjaeschke Please review my latest suggestion.

File: vars/stageStartOpenshiftBuild.groovy (hunk 1 above)
Comment (tjaeschke) [reply vars/stageStartOpenshiftBuild.groovy]: We will test it. Looks good to me.

**Comment by tjaeschke** [comment]
@michaelsauter The regex was a quickshot and I agree, we should revisit it after 2.0.

**Comment by michaelsauter** [comment]
@tjaeschke Can you please apply my suggestion then and merge?

**Comment by tjaeschke** [comment]
closes #159

**Pull request merged**